proposed/2023/NuGetAudit-without-nuget.org-as-packageSource.md

```diff
@@ -0,0 +1,221 @@
+# NuGetAudit without nuget.org as a package source
+<!-- Replace `Title` with an appropriate title for your design -->
+
+- Author: [zivkan](https://github.com/zivkan)
+- GitHub Issue: [12698](https://github.com/NuGet/Home/issues/12698)
+
+## Summary
+
+<!-- One-paragraph description of the proposal. -->
+This proposal adds a new `<auditSource>` to *NuGet.Config* files, allowing *NuGetAudit* to obtain known vulnerabilities without adding nuget.org as a package source.
```

Comment:
Will we add a new sub-verb to `dotnet nuget add` like `dotnet nuget add audit-source`? We have `dotnet nuget add source` today which allows CLI or scripted addition of package sources. Wonder if we should start with CLI support for manipulating audit sources.

Reply:
This is already addressed at the end of the document, in the "tooling" header under the "future possibility" section.

Also see https://github.com/NuGet/Home/pull/12918#discussion_r1356721787

Reply:
Oops, missed that. Thanks!